### PR TITLE
matcha-jp.com, tomshardware.com, vgmaps.com

### PR DESCRIPTION
**Pull request by DandelionSprout**
Example pages:

```
! https://matcha-jp.com/en/7823
matcha-jp.com##.c-post:has(:scope > div[class*="RelatedNative"])
! https://www.tomshardware.com/news/adblock-ublock-fake-google-chrome-browser-extensions,40422.html
tomshardware.com##.slot-top_of_sidebar
tomshardware.com##.slot-before_popular_box
tomshardware.com##.slot-after_popular_box
! https://www.vgmaps.com/Atlas/WiiU/index.htm
||vgmaps.com/Ad-*.htm
||vgmaps.com/Header.htm
||vgmaps.com/System/Main/Header-Sponsors.png
vgmaps.com##td[align="center"][width="728"]
```

### CODE REVIEW

File: dist/placeholder-buster.txt [hunk 1]
```diff
@@ -250,9 +250,13 @@ gadgethacks.com##.whtaph-floatbox
...
-tomshardware.*##.p-u-md-1-3.p-u-1:has(div[class*="block-ads"])
-tomshardware.*###in_article
-tomshardware.*###below_the_article
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/57
+tomshardware.com##.p-u-md-1-3.p-u-1:has(div[class*="block-ads"])
+tomshardware.com###in_article
+tomshardware.com###below_the_article
+tomshardware.com##.slot-top_of_sidebar
+tomshardware.com##.slot-before_popular_box
+tomshardware.com##.slot-after_popular_box
```
Comment (jspenguin2017) [inline dist/placeholder-buster.txt:259]: So the wildcard is no longer needed?

**Review by jspenguin2017 — CHANGES_REQUESTED** [review]

File: dist/placeholder-buster.txt (hunk 1 above)
Comment (DandelionSprout) [reply dist/placeholder-buster.txt:259]: `tomshardware.co.uk` appears to have been replaced by `tomshardware.com/uk`, so that is indeed correct.

**Review by jspenguin2017 — APPROVED** [review]

**Comment by jspenguin2017** [comment]
`https://matcha-jp.com/en/7823`

**Comment by jspenguin2017** [comment]
`https://www.tomshardware.com/news/adblock-ublock-fake-google-chrome-browser-extensions,40422.html`

**Comment by jspenguin2017** [comment]
`https://www.vgmaps.com/Atlas/WiiU/index.htm`

**Pull request merged**